### PR TITLE
chore: disable test, add iterations to existing

### PR DESCRIPTION
**Pull request by layershifter**
This PR makes few small changes to existing tests.

### CODE REVIEW

File: packages/fluentui/docs/src/examples/components/Button/Performance/ButtonOverridesMiss.perf.tsx [hunk 1]
```diff
@@ -167,4 +167,7 @@ const ButtonOverridesMissPerf = () => (
   </Provider>
 );
 
+ButtonOverridesMissPerf.iterations = 100;
+ButtonOverridesMissPerf.filename = 'ButtonOverridesMiss.perf.tsx';
```
Comment (layershifter) [inline packages/fluentui/docs/src/examples/components/Button/Performance/ButtonOverridesMiss.perf.tsx:171]: This should improve stability of these perf tests

File: packages/fluentui/e2e/tests/popupWithCloseInContent-test.ts [hunk 2]
```diff
@@ -9,7 +9,7 @@ describe('Popup With Close in the content and hover trigger', () => {
     await e2e.gotoTestCase(__filename, popupTrigger);
   });
 
-  it('A click on content should not close popup, a click button should close popup', async () => {
+  xit('A click on content should not close popup, a click button should close popup', async () => {
```
Comment (layershifter) [inline packages/fluentui/e2e/tests/popupWithCloseInContent-test.ts:12]: This test is unstable due a bug in `Popup`

File: packages/fluentui/e2e/tsconfig.json [hunk 3]
```diff
@@ -1,9 +1,10 @@
 {
   "extends": "../../../scripts/typescript/tsconfig.fluentui",
   "compilerOptions": {
-    "module": "esnext",
     "allowSyntheticDefaultImports": true,
-    "types": ["node", "jest", "webpack-env"]
+    "module": "esnext",
+    "types": ["node", "jest", "webpack-env"],
+    "skipLibCheck": true
```
Comment (layershifter) [inline packages/fluentui/e2e/tsconfig.json:7]: Should improve build speed a bit.

**Review by pompomon — APPROVED** [review]

File: packages/fluentui/e2e/tests/popupWithCloseInContent-test.ts (hunk 2 above)
Comment (layershifter) [reply packages/fluentui/e2e/tests/popupWithCloseInContent-test.ts:12]: https://github.com/microsoft/fluentui/issues/14800

**Comment by ecraig12345** [comment]
Merging this since the failures are in screener (not the edited tests) and therefore unrelated.

**Pull request merged**